Scripts/fetch-prebuilt-wda.js

```diff
@@ -0,0 +1,37 @@
+const path = require('path');
+const request = require('request-promise');
+const { asyncify } = require('asyncbox');
+const { logger, fs, mkdirp } = require('appium-support');
+const { exec } = require('teen_process');
+
+const log = logger.getLogger('WDA');
+
+async function fetchPrebuiltWebDriverAgentAssets () {
+  const tag = require('../package.json').version;
+  const downloadUrl = `https://api.github.com/repos/appium/webdriveragent/releases/tags/v${tag}`;
+  log.info(`Getting WDA release ${downloadUrl}`);
+  const releases = await request.get(downloadUrl, {
+    headers: {
+      'user-agent': 'node.js',
+    },
+    json: true,
+  });
+  const webdriveragentsDir = path.resolve(__dirname, '..', 'webdriveragents');
+  log.info(`Downloading assets to: ${webdriveragentsDir}`);
+  await fs.rimraf(webdriveragentsDir);
+  await mkdirp(webdriveragentsDir);
+  for (const asset of releases.assets) {
+    const url = asset.browser_download_url;
+    log.info(`Downloading: ${url}`);
+    // wget never seems to exit successfully so just ignore non-zero status code
+    try {
+      await exec('wget', [url, '.'], {cwd: webdriveragentsDir});
```

Comment:
I would rather use requests module for this purpose. Same thing we do in https://github.com/appium/appium-base-driver/blob/da44da41ddf18448179254997805f2f8265f1596/lib/basedriver/helpers.js#L205